tensorboard/webapp/BUILD

```diff
@@ -207,13 +207,27 @@ tf_web_library(
 
 # A Vulcanized html binary for the complete app (both Angular and Polymer parts)
 tensorboard_html_binary(
-    name = "index",
+    name = "index_binary",
     input_path = "/index_polymer3.inlined.html",
     js_path = "/index.js",
-    output_path = "/index.html",
+    output_path = "/index_binary.html",
```

Comment:
Maybe give this a name that more clearly indicates the difference
compard to `index.html` (which *isn’t* that one contains more or less
binary content)? e.g., `/index.no_resource_digests.html` following our
existing patterns, but I’m open to suggestions.